lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -1011,4 +1015,25 @@ std::string SPIRVToLLVMDbgTran::findModuleProducer() {
   return "spirv";
 }
 
+Optional<DIFile::ChecksumInfo<StringRef>>
+SPIRVToLLVMDbgTran::ParseChecksum(StringRef Text) {
+  // Example of checksum string:
+  // "//__CSK_MD5:7bb56387968a9caa6e9e35fff94eaf7b"
+  Optional<DIFile::ChecksumInfo<StringRef>> CS;
+  auto KindPos = Text.find(SPIRVDebug::ChecksumKindPrefx);
+  if (KindPos != StringRef::npos) {
+    auto ColonPos = Text.find(":", KindPos);
+    KindPos += string("//__").size();
```

Comment:
I suggest to just drop `//__` from the string instead of calculating the current position to parse. It can be a good in combination with [ltrim](https://llvm.org/doxygen/classllvm_1_1StringRef.html#a9db1f51e975fa287ad0fd6a44c567974), for this algorithm instead of finding ":" in the string we can just call `ltrim(':')` on a string with dropped `//__`.

Reply:
Is this what you meant @MrSidims?

Reply:
Yeap. Since we can't drop `//__` prefix it's LGTM.